Hestia/Hestia/backend/NetworkHandler.cs

```diff
@@ -10,19 +10,19 @@ namespace Hestia.backend
 {
     public class NetworkHandler
     {
-        private string ip;
+        private string address; // address including connection method
         private int port;
```

Comment:
Not asking for port anymore so could use default here

Reply:
@daangroot will share his thoughts about this.

Reply:
`address` contains the port after #223 is merged.